README.md

```diff
@@ -45,3 +45,9 @@ Alternatively, each app can be opened individually.
 ## Licensing and Use
 
 Azure Digital Twins Samples are [MIT Licensed](./LICENSE.md).
```

Comment:
Did you mean to change this as well?  There might be others too I didn't look carefully.

Reply:
Thanks! Corrected.